assets/src/settings-page/style.css

```diff
@@ -413,7 +413,6 @@ li.error-kept {
 }
 
 #paired-url-structure .amp-paired-url-examples summary {
-	display: inline-block;
```

Comment:
Humm. I had added this 5 months ago for some reason: https://github.com/ampproject/amp-wp/commit/7354ccf525465e3f8a88f85fc3717cac495fd33d

Reply:
This behavior of the `summary` element is quite odd to me and I wasn't aware of it before. But it seems it's a known limitation/quirk ([see the last paragraph](https://css-tricks.com/two-issues-styling-the-details-element-and-how-to-solve-them/#inline-all-the-things)).